### ISSUE TITLE
Arbitrary index used for padding

### ISSUE THREAD

**Issue by kawu** [ShenMinX/point-gen#2]
In `encode.py`, initialization method, the padding index is hard-coded:
```python
self.embedding = torch.nn.Embedding(num_embeddings = vocab_size, embedding_dim = embed_size, padding_idx=3)
```
Is the `3` value special somehow?  This code doesn't look too safe (and hard-coding should be avoided in general...).
